### DISCUSSION TITLE
An username already taken but i want that

### DISCUSSION THREAD

**Discussion by ulf-frostypaw** [community/community#71728] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

The username @keso was taken by someone who has not been active for 12 years. It no longer has any activity, at least not public. Please I want to take the username for myself. I hope someone can help me. QWQ

**Comment by wuleninja** [comment]
I have the exact same issue, with the name of the company I'm working for. For now, the only solution I seem to have is going to GitLab :/

**Comment by keso** [comment, marked as answer]
I might not be active with contributing, but I still need my account to access repos once in a while. 🤷🏽‍♂️ And who knows, I might contribute some stuff in the future as well. So I'll hold on to it.

**Reply by ulf-frostypaw** [reply]
OH. Well, I guess I can't change things.

**Comment by ghostinhershell** [comment]
Hey there @ulf-frostypaw, 

I'm glad we were able to get a resolution here but, for future reference, keep in mind that not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.

If the username you want has already been claimed, consider other names or unique variations. Using a number, hyphen, or an alternative spelling might help you identify a desirable username that's still available. Cheers!
